tools/wptrunner/wptrunner/executors/executorwebdriver.py

```diff
@@ -311,6 +312,15 @@ def set_user_verified(self, authenticator_id, uv):
         return self.webdriver.send_session_command("POST", "webauthn/authenticator/%s/uv" % authenticator_id, uv)
 
 
+class WebDriverSPCTransactionsProtocolPart(SPCTransactionsProtocolPart):
+    def setup(self):
+        self.webdriver = self.parent.webdriver
+
+    def set_spc_transaction_mode(self, mode):
+        mode_json = {"mode": mode}
```

Comment:
This isn't JSON yet, maybe call it `body` or `mode_dict`?

Reply:
Done.